README.md

```diff
@@ -674,7 +674,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [SX1278]: https://github.com/susu/sx1278
 [RFM69]: https://github.com/lolzballs/rfm69
 [LS010B7DH01]: https://github.com/byronwasti/ls010b7dh01
-[MAX31855]: https://github.com/mbacch/max31855
+[MAX31855]: https://github.com/cs2dsb/max31855.rs
```

Comment:
Shouldn't this be <https://crates.io/crates/max31855>? I can't remember if we're keen on linking to crates.io over Github.